readme.md

```diff
@@ -1,25 +1,40 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/262842523/22.2.6%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T888962)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-*Files to look at*:
 
-* [Module.cs](./CS/EFCore/NonPersistentEditEF/NonPersistentEditEF.Module/Module.cs)
-* [ProductView.cs](./CS/EFCore/NonPersistentEditEF/NonPersistentEditEF.Module/BusinessObjects/ProductView.cs)
+# XAF - How to edit a collection of persistent objects linked to a non-persistent object
+
+When a [non\-persistent object](https://docs.devexpress.com/eXpressAppFramework/116516/concepts/business-model-design/non-persistent-objects) contains a collection of persistent business objects, you may need to edit linked objects right in the list view either in in-place or in `ListViewAndDetailView` mode. Also, if a linked persistent object is edited in a popup detail view, you may need to see changes in the source list view after they are saved.
+
+This example demonstrates how you can perform these tasks in XAF applications.
 
-# How to edit a collection of Persistent Objects linked to a Non-Persistent Object
+> [!WARNING]
+> We created this example for demonstration purposes and it is not intended to address all possible usage scenarios with it.
+> If this example does not have certain functionality or you want to change its behavior, you can extend this example as needed. Please note that this can be a complex task that requires good knowledge of XAF: [UI Customization Categories by Skill Level](https://www.devexpress.com/products/net/application_framework/xaf-considerations-for-newcomers.xml#ui-customization-categories). You will likely need to research how our components work under the hood. Refer to the following help topic for more information: [Debug DevExpress .NET Source Code with PDB Symbols](https://docs.devexpress.com/GeneralInformation/403656/support-debug-troubleshooting/debug-controls-with-debug-symbols).
```

Comment:
```suggestion
> If this example does not have certain functionality or you want to change its behavior, you can extend this example. This can be a complex task that requires good knowledge of XAF: [UI Customization Categories by Skill Level](https://www.devexpress.com/products/net/application_framework/xaf-considerations-for-newcomers.xml#ui-customization-categories), and you may need to research how our components work. Refer to the following help topic for more information: [Debug DevExpress .NET Source Code with PDB Symbols](https://docs.devexpress.com/GeneralInformation/403656/support-debug-troubleshooting/debug-controls-with-debug-symbols).
```